.circleci/config.yml

```diff
@@ -182,6 +182,7 @@ jobs:
       - run:
           name: Run tests
           command: bundle exec fastlane test_ios
+          no_output_timeout: 30m
```

Comment:
I thought about this, cause that's the new thing that was making iOS 15 tests fail after I changed CI to use the new sims.
I guess as long as they're not taking forever this makes sense.

Reply:
@NachoSoto Hah, I just commented on another PR- CI seems to be restarting the sims and it's taking forever 🙃